weather/data_funcs.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+# ===================================================================
+# Useful functions
+# ===================================================================
+
+
+def isfloat(value):
+    """
+    This function checks if a string can be converted to a float.
+
+    Parameters:
+    -----------
+    value : string
+
+    Returns:
+    --------
+    boolean
+    """
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def to_float(dataframe, column):
```

Comment:
Why don't you use to_nemeric, astype, or infer_objects? See [this](https://stackoverflow.com/questions/15891038/change-data-type-of-columns-in-pandas) discussion on changing pandas columns to floats. As someone who has 'reinvented the wheel' many times, I can tell you it's not worth it.